index.bs

```diff
@@ -1368,6 +1419,12 @@ Elected Groups Vacated Seats</h5>
 			and the maximum number corresponds to all unoccupied seats.
 			Except for the number of available seats and the length of the terms,
 			the <a href="#AB-TAG-elections">usual rules for Advisory Board and Technical Architecture Group Elections</a> apply.
+
+		<li>
+			If seats are vacated due to a successful [=vote of no confidence=],
```

Comment:
I think that you want to set a threshold for triggering an election, not just have this for the no confidence vote.  The bodies will likely function with a few gaps, but there is a critical point at which the groups are non-viable.  I might suggest 2/3 of the target size as that threshold.

```suggestion
			If vacancies reduce
			the number of [=AB=] participants to 6 or fewer
			or the number of [=TAG=] participants to 7 or fewer,
```

Reply:
That seems like a separate discussion. The previous bullet point in this list already deal with the case of seats being vacated for reasons other than a successful vote of no confidence, by letting the chair decide whether to run an special election or not. Your proposal would make it no longer a choice by the chair, which may or may not be a good idea, but is a separate idea. If you want to pursue that, could I ask you to open a separate issue?

Reply:
I sort of agree, but I don't think that you can separate that out completely.  You are introducing new ways in which the membership of these groups can drop to critically low levels.  Though we don't expect this process to be used often, in combination with existing stuff, you have increased the risk of the groups being understaffed.  Staffed to zero is just one special case.

Reply:
What threshold should trigger an automatic election is an interesting question, but probably should be discussed as its own issue. What we can say for sure is that a threshold of zero needs to trigger an election. :) So how about we write the PR here for the case of zero, and we can open an issue about possibly raising that number higher.

Reply:
I think that works. We do have a means to deal with a low (non-zero) number of participants (chair discretion) but it seems worth discussing whether an automatic (non-zero) threshold would be better. I'm landing the zero threshold change, let's discuss the rest a https://github.com/w3c/process/issues/1032

```suggestion
			If vacancies reduce the number of [=AB=] or [=TAG=] participants to 0,
```